app/services/experimentation/export.rb

```diff
@@ -76,6 +83,56 @@ def experimental_communications(notifications)
       end
     end
 
+    def followup_results(patient, tgm, notifications)
+      if experiment.experiment_type == "current_patients"
+        active_patient_followups(patient, notifications)
+      elsif experiment.experiment_type == "stale_patients"
+        stale_patient_followups(patient, tgm)
+      else
+        raise ArgumentError("No followup logic available for experiment type #{experiment.experiment_type}")
+      end
+    end
+
+    def active_patient_experimental_appointment_dates(patient, notifications)
+      if notifications.any?
+        notifications.map(&:subject).pluck(:scheduled_date).uniq
+      else
+        # control patients don't have notifications
+        experiment_date_range = (experiment.start_date..experiment.end_date)
+        patient.appointments.where(scheduled_date: experiment_date_range)
+          .where("device_created_at < ?", experiment.start_date.beginning_of_day)
+          .order(:scheduled_date)
+          .pluck(:scheduled_date)
+          .uniq
+      end
+    end
+
+    def active_patient_followups(patient, notifications)
+      appointment_dates = active_patient_experimental_appointment_dates(patient, notifications)
+      appointment_dates.map.each_with_index do |appointment_date, index|
+        adjusted_index = index + 1
+        followup_date_range = ((appointment_date - FOLLOWUP_START)..(appointment_date + FOLLOWUP_CUTOFF))
```

Comment:
These date ranges are a bit arbitrary, but there is no clear relationship between an appointment and a blood pressure/encounter. The ten day cutoff is something I believe Reena suggested. The three days before... that's for the case that a patient comes in before their appointment date.